src/importexport/musicxml/internal/musicxml/importmxmlpass2.cpp

```diff
@@ -5295,7 +5298,7 @@ Note* MusicXMLParserPass2::note(const QString& partId,
             note->setColor(noteColor);
         }
         setNoteHead(note, noteheadColor, noteheadParentheses, noteheadFilled);
-        note->setVisible(printObject); // TODO also set the stem to invisible
+        note->setVisible(printObject && !headInvisible); // TODO also set the stem to invisible
```

Comment:
`hasHead && printObject` (see abve)

Reply:
`note->setVisible` will make whole note invisible if param is false? Is this expected behavior in case we need only notehead to be invisible?

Reply:
note == notedhead.